lib/tasks/dev.rake

```diff
@@ -79,6 +79,8 @@ namespace :dev do
 
   desc 'Create in-person enrollments for N random users'
   task random_in_person_users: [:environment, :random_users] do
+    usps_request_delay_ms = (ENV['USPS_REQUEST_DELAY_MS'] || 0).to_i
+    usps_request_delay_ms.freeze
     num_users = (ENV['NUM_USERS'] || 100).to_i
     pw = 'salty pickles'
     unless ENV['PROGRESS'] == 'no'
```

Comment:
Commenting here because I can't on line 100 below. Is there any reason not to change the behavior of this script so that for each user/enrollment pair we use one transaction rather than using one transaction for all N users and enrollments? Like this:

```
    (0...num_users).each do |n|
      InPersonEnrollment.transaction do
```

That way, in the event of unexpected errors from USPS, the script will fail at whatever user count it made it to rather than rolling back the entire transaction. Seems like the main consequence would be that load testing would ramp up gradually as the script ran instead of going from 0 load to `num_users` load at the end and that doesn't seem like a bad thing to me.

Reply:
I included this in the original so that the behavior worked similarly to the pre-existing command for generating users. I think this does make the script's behavior simpler to reason about.

That said, I think your suggestion here is worth implementing since it could reduce the time spent generating enrollments.